crates/js-sys/src/lib.rs

```diff
@@ -483,6 +484,30 @@ extern "C" {
     pub fn value_of(this: &Boolean) -> bool;
 }
 
+impl From<bool> for Boolean {
+    fn from(b: bool) -> Boolean {
+        Boolean { obj: JsValue::from(b) }
```

Comment:
This looks a bit like dark magic :)

I didn't know these types have an internal `obj` field and I guess it might change in the future. Would `JsValue::from_bool(b).unchecked_into()` maybe look better?

Reply:
Good point!